repos/spack_repo/builtin/packages/py_llvmlite/package.py

```diff
@@ -39,7 +42,8 @@ class PyLlvmlite(PythonPackage):
     depends_on("python@:3.9", when="@0.36:0.37", type=("build", "run"))
 
     # https://github.com/numba/llvmlite#compatibility
-    depends_on("llvm@15", when="@0.44:")
+    depends_on("llvm@20", when="@0.45:")
+    depends_on("llvm@15", when="@0.44")
```

Comment:
Per compatibility table:
```suggestion
    depends_on("llvm@15:16", when="@0.44")
```